src/test/java/assistinator/StorageTest.java

```diff
@@ -0,0 +1,34 @@
+package assistinator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+public class StorageTest {
```

Comment:
Nice naming of the tests to explicate exactly what they are for! It would be nice to include javadocs to make the tests understandable to contributors. But overall, this is well-formatted and suits the coding conventions! 👍